### ISSUE TITLE
Nexus 5x Oreo don't sync to LG G Watch AsteroidOS

### ISSUE THREAD

**Issue by c1p81** [AsteroidOS/AsteroidOSSync#51]
As the subject the app does not open (crash on start)
Thanks
Luca

**Comment by c1p81** [comment]
The version from F-Droid chrash. The version compiled from GitHub works.

**Comment by FlorentRevest** [comment]
Right, it's probably a translation error in the version of F-Droid. What language do you use? Could you try the f-droid version with your phone's language set to english?

**Comment by moWerk** [comment]
Outdated, fell free to make a new issue if still relevant.
